runatlantis.io/docs/using-atlantis.md

```diff
@@ -1,19 +1,40 @@
 # Using Atlantis
 
-Atlantis currently supports three commands that can be run via pull request comments:
+Atlantis handle commands via pull request comments.
+![Help Command](./images/pr-comment-help.png)
+
+::: tip
+You can use following executable names.
+* `atlantis help`
+  * `atlantis` is executable name. You can configure by [Executable Name](/docs/server-configuration.html#executable-name)
+* `run help`
+  * `run` is a global executable name.
+* `@GithubUser help`
+  * `@GithubUser` is the VCS host user which you connected to Atlantis by user token.
+:::
```

Comment:
IMO it is better to drop `run help` feature...

Reply:
It's your opinion to drop the `run help` feature because it's a hidden function?

Wouldn't you want to add the `run help` feature because it's a hidden function?

Reply:
I want to remove `run help` feature. so remove this doc.

https://github.com/runatlantis/atlantis/pull/2808/commits/ee9777a91bac254ea58b13bb9166e8770fe33964

Reply:
Why remove the `run help` feature?

Reply:
I guess no one use `run apply` and almost users doesnot know this feature. Not to execute it untentionally by just comment it, I think it is better to drop it.

Reply:
It's possible people still use it. We don't collect analytics for this. It's probably best not to introduce a breaking change, no?

Reply:
True. Generally, we should not do a breaking change. it must be introduced into major update and so on. In this PR, just add docs and not to drop it and so on. Currently, we already provide this feature, so I add docs 👌 

https://github.com/runatlantis/atlantis/pull/2808/commits/3358e38ca0a320acdeeab9eda331f40c8a8c583d